wasm-engines/docker/bench-base.Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:19.10
```

Comment:
This image seems to be mostly about python, why not base it on `circleci/python:3.7` ?

Reply:
I'd opt not to change the base image at this point because it saves ~30s of build time at the cost of breaking images that depend on it (necessitating significant time to fix them and rebuild/test).  I also prefer to have the base images in repos that we control.